CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ OPTION(WANT_VST_NOWINE	"Include partial VST support (without wine)" OFF)
 OPTION(WANT_WINMM	"Include WinMM MIDI support" OFF)
 OPTION(WANT_QT5		"Build with Qt5" OFF)
 OPTION(WANT_DEBUG_FPE	"Debug floating point exceptions" OFF)
+OPTION(WANT_DEBUG_PERFLOG "Log task performance" ON)
```

Comment:
Should the default value be `OFF`, like FPE debugging?

Reply:
OFF sounds right to me, too.